src/Helper/StringHelper.php

```diff
@@ -26,4 +26,15 @@ public static function splitString(string $string, int $maxLength): array
         }
         return [$full];
     }
+
+    public static function findStringInArray(string $needle, array $haystack): ?string
```

Comment:
isnt that kind of, what http://php.net/manual/de/function.array-search.php is for?

Reply:
no it is not the same function